src/mp-instance.js

```diff
@@ -1523,9 +1523,8 @@ function createKitBlocker(config, mpInstance) {
 
 function runPreConfigFetchInitialization(mpInstance, apiKey, config) {
     mpInstance.Logger = new Logger(config);
-    mpInstance._Store = new Store(config, mpInstance);
+    mpInstance._Store = new Store(config, mpInstance, apiKey);
```

Comment:
This feels like a big change. What's the rationale here?

Reply:
Also, since line 1528 previously was just setting the apiKey on the store, it seemed like it should always have just been passed into the Store.